examples/inverse/plot_resolution_metrics.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
+#
+# License: BSD (3-clause)
+"""
+Plot PSFs and CTFs with corresponding resolution metrics.
+
+Compare metrics localisation error and spatial extent with the
+corresponding PSFs and CTFs.
+"""
```

Comment:
@olafhauk see how the headers of examples should be formatted. Authors below docstring, title with === under it etc.